src/components/App.tsx

```diff
@@ -309,8 +326,17 @@ class App extends React.Component<any, AppState> {
       } else {
         scene = await loadScene(null);
       }
+
       if (scene) {
         this.syncActionResult(scene);
+        if (scene.appState) {
```

Comment:
This will only run on single user mode. Probably move this to the end of this function so will even run on collaboration mode as well.